javascript-practice/src/pages/dashboard.html

```diff
@@ -95,6 +97,99 @@ <h2 class="user-dashboard-heading">Users Dashboard</h2>
           </div>
         </div>
       </section>
+      <div class="popup-overlay">
+        <div id="popupContainer" class="popup-container">
+          <div class="flex align-center popup-heading-box">
+            <h2 class="popup-heading">Add User</h2>
+            <div id="closePopup" class="close-popup-box">
+              <img src="../assets/icons/alert-circle.svg" alt="" />
+            </div>
+          </div>
+          <form action="" class="popup-form-box flex-column">
+            <input
+              class="input popup-input"
+              type="text"
+              placeholder="User ID *"
+              name=""
+              id=""
+            />
+            <div class="flex popup-area-form-box">
+              <input
+                class="input popup-input"
+                type="text"
+                placeholder="First Name *"
+                name=""
+                id=""
+              />
+              <input
+                class="input popup-input"
+                type="text"
+                placeholder="Last Name *"
+                name=""
+                id=""
+              />
+            </div>
+            <div class="flex popup-area-form-box">
+              <input
+                class="input popup-input"
+                type="text"
+                placeholder="Email ID *"
+                name=""
+                id=""
+              />
+              <input
+                class="input popup-input"
+                type="text"
+                placeholder="Mobile No"
+                name=""
+                id=""
+              />
+              <div class="input popup-input flex align-center space-between">
+                <select name="" id="" class="popup-select">
+                  <option class="popup-option" value="" disabled selected>
+                    Select Role Type
+                  </option>
+                  <option class="popup-option" value="">Supper Admin</option>
```

Comment:
List of role should be generated by JS code because it is dynamic, not fixed values

Reply:
I'll update it later